config/settings.yml

```diff
@@ -14,6 +14,12 @@ logingov:
   client_cert_path: spec/fixtures/sign_in/oauth.crt
   client_key_path: spec/fixtures/sign_in/oauth.key
   oauth_url: https://idp.int.identitysandbox.gov
+  inherited_proofing:
```

Comment:
I still would make these completely separate, not in the login_gov namespace. So I guess just at the `inherited_proofing` level. The alternative could be to push the sign_in service logingov stuff into a namespace to look like:

logingov:
  sign_in:
    oauth_url....
    ...
  inherited_proofing:
    oauth_url....
    ...

I guess if we wanted to do that we could keep this as is and add a story to do that for the sign_in service settings